src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  margin: 0 auto;
+  width: 100%;
+  background-color: $bg-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  z-index: 1000;
+
+  &__container {
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
```

Comment:
The padding property in `.header__container` seems to be too large for a header. It could be reduced to something like `10px` or `20px`.